packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -51,9 +52,13 @@ const ProductDetail = () => {
     const einstein = useEinstein()
     const toast = useToast()
     const navigate = useNavigation()
-    const [productSetSelection, setProductSetSelection] = useState({})
-    const childProductRefs = React.useRef({})
     const customerId = useCustomerId()
+
+    /****************************** Sets and Bundles *********************************/
+    const [childProductSelection, setChildProductSelection] = useState({})
+    const [childProductOrderability, setChildProductOrderability] = useState({})
```

Comment:
I have it at the PDP level because the sibling `ProductView` components need to be able to share data between each other, data being the availability status of the child product in order to determine if we disable the add to cart button at the top level bundle `ProductView` component. There may be better ways to do this, but I facilitate the data between the `ProductView` components through the PDP parent component.

Reply:
> ...  I facilitate the data between the ProductView components through the PDP parent component

Ah got it, sorry my bad.. that explanation makes it clear what's going on 👍